### ISSUE TITLE
Diablo IV.exe Location

### ISSUE THREAD

**Issue by Plinpod** [cjbrigato/d4-bnet-mitm#5]
Hey!

Is this dependent that the Diablo IV.exe location is at C:\Program Files (x86)\Diablo IV?

**Comment by cjbrigato** [comment]
Not at all, you have to launch the binary by yourself with proper command line argument to override the auth server from `prod.actual.battle.net` to `localhost:yourport` or you have yo enforce proper routing via you hosts file

**Comment by Plinpod** [comment]
oh gotcha, that makes sense

**Comment by advocaite** [comment]
what is the command to do that?
